src/sbt-test/sbt-sass-compiler/can-compile-if-dart-sass-executable-missing/build.sbt

```diff
@@ -0,0 +1,39 @@
+import java.nio.file.{Files, Paths}
+import scala.sys.process._
+
+lazy val sassEmbeddedHostVersion = "4.0.2"
+lazy val tempDir        = Paths.get(s"/tmp/de.larsgrefer.sass.embedded.connection.BundledPackageProvider/${sassEmbeddedHostVersion}")
+lazy val executableFile = Paths.get(s"/tmp/de.larsgrefer.sass.embedded.connection.BundledPackageProvider/${sassEmbeddedHostVersion}/dart-sass/1.83.4/dart-sass/sass")
+
+lazy val findSassEmbedded   = taskKey[Unit]("Search for dart-sass executable")
+lazy val deleteSassEmbedded = taskKey[Unit]("Delete dart-sass directory")
+lazy val createSass         = taskKey[Unit]("Create a sass file to trigger rebuild")
+
+lazy val root = (project in file("."))
+  .enablePlugins(SbtWeb)
+  .settings(
+    deleteSassEmbedded     := {
+      if (Files.exists(tempDir)) {
+        println(s"Deleting $tempDir ...")
+        s"rm -rf $tempDir".! // Works when outside of scripted sandbox
```

Comment:
with this I think we should only delete the executableFile to properly recreate the bug - where if we delete the whole tempDir path above then it should work even without out fix of retrying I would have thought

Reply:
I would maybe fold deleteSassEmbedded and findSassEmbedded into one task so we get a test somewhat like:
```
assets
recreateBug
assets
```

Reply:
I changed my thoughts on above and tweaked the example test script a bit 😅 take that as all optional knit picking design stuff that could be not done or tweaked further, main bit is to only delete the sass executable

One thing you could do maybe too now I think about it in the recreateBug is we could possibly call `Using(SassCompilerFactory.bundled())` ourselves in the recreateBug task and check that we get a error - so we know if at some point in future we don't need our code anymore because it is being fixed upstream instead

Reply:
where recreateBug could be something like:
```
val sassBinary = new File(...)
if(IO.notExists(sassBinary)) sys.exit("sass binary does not exist, does path need updating?")
IO.delete(sassBinary)
Using(SassCompilerFactory.bundled()) { _ => 
  sys.exit("deleting sass binary didn't cause an error, does bug still exist?")
}
// successfully recreated bug
```